### ISSUE TITLE
random lockups on esp8266

### ISSUE THREAD

**Issue by dutchy1966** [ChuckBell/MySQL_Connector_Arduino#2]
Hi,
I have found reference in some places about an issue I have with the mysql library. The issue seems to related to random lockups of the esp8266 board when executing queries. Where ost people reporting this issue contribute this to the library itself, I have my doubts.
In my sketch, which I use unaltered for months, and has been rock stable for ages I can see the issue when it is compiled using de arduino esp libraries >= 2.0.0
Up until recently I have been developing with esp library 1.6.5 and there has never been any sign of the problem. I also like to add I have this problem already with 1.0.4ga version. Therfore it does not seem to be an issue of the newer releases of the mysql library.
So basically:
1.6.5. esp libraries --> stable, 2.0+ esp libraries --> random lockups.
I have placed this here because comments are no longer allowed on the blog.
I hope this sheds some light on the issue...and someone has a clue to what is going on.

**Comment by ChuckBell** [comment]
Thanks for the information.

I closed the 1.0.X blogs because they are no longer relevant for comments on the newest code. For new comments and questions, you can post on http://drcharlesbell.blogspot.com/2016_01_01_archive.html.

The best place to post comments is on http://forums.mysql.com/list.php?175.

I will add this information to the documentation FAQ section so you'll see it in the next release. Until then, I'll leave this issue open.
